xmppserver/src/main/java/org/jivesoftware/openfire/http/HttpBindManager.java

```diff
@@ -93,7 +93,7 @@ public final class HttpBindManager implements CertificateEventListener {
     public static final SystemProperty<Integer> HTTP_BIND_SECURE_PORT = SystemProperty.Builder.ofType(Integer.class)
         .setKey("httpbind.port.secure")
         .setDynamic(true)
-        .setDefaultValue(7444)
+        .setDefaultValue(7443)
```

Comment:
I believe that the problem being corrected here is not in the upstream Openfire source, which must mean that it gets introduces somewhere in this PR.

Rather than creating problem in one commit and needing another commit to fix it, please squash this commit with the one that introduces the problem (or edit the commit that introduces the issue, and remove the commit that fixes it). That way, it is as if it never happened (and this won't be in our source history).